demos/python_demos/requirements.txt

```diff
@@ -1,5 +1,6 @@
 --extra-index-url https://download.pytorch.org/whl/cpu 
 git+https://github.com/mzegla/optimum-intel.git@infer_request_pool
+# used a fork to provide support for multi concurrency
 onnx
 pillow
 optimum[diffusers]
```

Comment:
```suggestion
optimum[diffusers]
tritonclient[grpc]
```

Reply:
there is added a separate client_requirements.txt to show which dependencies are needed by the client and which to download the model.